src/main/java/net/fabricmc/loom/task/NonRemappedJarTaskConfiguration.java

```diff
@@ -52,7 +52,7 @@ public void configure() {
 		project.getTasks().named(JavaPlugin.JAR_TASK_NAME, Jar.class).configure(task -> {
 			task.dependsOn(processIncludeJarsTask);
 			// Use JarNester to properly add jars and update fabric.mod.json
-			task.doLast(new NestJarsAction(processIncludeJarsTask.flatMap(NestableJarGenerationTask::getOutputDirectory)));
+			task.doLast(new NestJarsAction(project.fileTree(processIncludeJarsTask.flatMap(NestableJarGenerationTask::getOutputDirectory))));
```

Comment:
You might want to add a jar file extension filter like in the old one but I don't know if it actually matters 🤔